spinalcordtoolbox/scripts/sct_process_segmentation.py

```diff
@@ -193,6 +209,16 @@ def get_parser():
         help="Extent (in mm) for the mask used to compute morphometric measures. Each slice covered by the mask is "
              "included in the calculation. (To be used with flag '-pmj' and '-pmj-distance'.)"
     )
+    optional.add_argument(
+        '-normalize',
+        metavar=Metavar.list,
+        action=SeperateNormArgs,
+        nargs="+",
+        help="R|Normalize CSA values ('MEAN(area)'). Two models are available: \n1. sex, brain-volume, thalamus-volume."
+        " \n2.sex, brain-volume. \nSpecify each value for the subject after the corresponding predictor.\n"
+        "Example: -normalize sex 0 brain-volume 960606.0 thalamus-volume 13942.0 \n"
+        "brain-volume and thalamus-volume are in mm^3. For sex, female: 0, male: 1."
```

Comment:
Truthfully, I'm not super familiar with CSA normalization (how it's typically done, the state of the art, how to do it "correctly"). When I read [Julien's comments](https://forum.spinalcordmri.org/t/csa-normalisation/113/2) on this previous forum post (about controversy, lack of consensus, risk of adding additional noise), it gives me the feeling that it would be good to provide a little more detail about the models themselves.

If I were a user, I would be curious about e.g. the subjects and methods that were used to compute these models. That way, if I were citing this feature, I could be transparent about which predictors were used to normalize the CSA values.

Could we add something like "These models were computed using the [uk-biobank-dataset] comprised of [x subjects] and [blah blah]. For more details on the subjects and methods used to create the models, go to [REF]." to the help description?

Reply:
I'm also wondering if there any risk that `-normalize` could be used improperly, say if the user applies it blindly to their data without thinking carefully of the implications. 

If so, maybe we could add a warning or disclaimer to the help description, too. For example, something like "Given the lack of consensus surrounding CSA normalization, please use this feature at your own discretion. We recommend thoroughly reviewing the literature on this topic before applying normalization to your data." 

(Or, would that be impolite to suggest?)

Reply:
You probably know this area much, much better than I do, so I would appreciate hearing your perspective on this, @sandrinebedard! :slightly_smiling_face:

Reply:
I added a more thorough description and the warning we talked about, can't put the link yet, I can maybe open an issue about this. Does it look ok to you @joshuacwnewton and @jcohenadad ?

Reply:
https://github.com/spinalcordtoolbox/spinalcordtoolbox/blob/8db927e4f786baa0f8ffd8eda160a44dd922c1f5/spinalcordtoolbox/scripts/sct_process_segmentation.py#L217-L224

I have just a few nitpicky comments:

* I think that the last sentence could maybe be removed ("The model has not been tested on other contrasts than T1w and on pathological patients."), as it seems redundant with the first sentence ("The models were generated from 804 healthy participants from 48 to 80 years old from the UK Biobank dataset on T1w brain images.")
* The first sentence could maybe be reworded a little for clarity: "The models were generated using T1w brain images from 804 healthy (non-pathological) participants ranging from 48 to 80 years old, taken from the UK Biobank dataset."
* I think that we should fill in the [REF] with a link to something, even if it's just temporary until a paper is finalized. (Is there a GitHub repo with a README.md that can be updated with the details, for example?) 
* I'm also wondering if we could maybe use newlines or indentations to make the paragraphs a little more clear, visually. Right now it looks a little bit like a "wall of text" -- there are a couple of different sections (argument description, usage, example, model description, disclaimer) and I find it a little tricky to parse the info.

Reply:
> Is there a way to fill in the [REF] with a link to something, even if it's just temporary until a paper is finalized? (Is there a GitHub repo with a README.md that can be updated with the details, for example?)

@joshuacwnewton : would this be good? https://github.com/sct-pipeline/ukbiobank-spinalcord-csa/blob/master/README.md

Reply:
does this look better?
 (in 
https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/3515/commits/52dc4a61dded4999899f5f04102387062e2056d8)